src/domain/enrolment/enrolmentForm/ValidationSchema.tsx

```diff
@@ -55,10 +55,12 @@ export default function getValidationSchema({
           // The used validation error message will be the same for both the fields.
           // This is also preventing negative param.max to occur in validation.
           if (
-            !sizePair ||
-            sizePair < 0 ||
-            sizePair < minGroupSize ||
-            sizePair > maxGroupSize
+            minGroupSize &&
+            maxGroupSize &&
```

Comment:
Is it possible to define only `minGroupSize` or `maxGroupSize` but leave the other undefined? If that is possible, then should that case also be taken into account here?